tests/MapperTestUtility.php

```diff
@@ -0,0 +1,196 @@
+<?php
+/**
+ * ownCloud - mediametadata
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ *
+ * @author Jalpreet Singh Nanda (:imjalpreet) <jalpreetnanda@gmail.com>
+ * @copyright Jalpreet Singh Nanda (:imjalpreet) 2016
```

Comment:
You need to keep the original author and copyright. You were not the original author of this class.
Please review the other classes as well, if you've imported blocks of code, you need to keep the author and copyright in the code.